docs/RELEASE-NOTES.rst

```diff
@@ -37,6 +37,10 @@ Bug Fixes
 * :issues:`649` - Route annotation profiles are no longer ignored.
 * :cccl-issue:`214` - Keys and certificates are now installed onto the managed partition.
 
+Limitations
+```````````
+* Can not apply app-root and url-rewrite annotations to the same resource see: https://github.com/F5Networks/k8s-bigip-ctlr/issues/675
```

Comment:
Nit: Can not should be cannot, and I would put a semicolon between "resource; see".